frontend/device/input.lua

```diff
@@ -1393,23 +1322,15 @@ function Input:inhibitInput(toggle)
             self.handleKeyBoardEv = self.handlePowerManagementOnlyEv
         end
         -- And send everything else to the void
-        if not self._oasis_ev_handler then
-            self._oasis_ev_handler = self.handleOasisOrientationEv
-            self.handleOasisOrientationEv = self.voidEv
-        end
         if not self._abs_ev_handler then
             self._abs_ev_handler = self.handleTouchEv
             self.handleTouchEv = self.voidEv
         end
-        if not self._msc_ev_handler then
-            if not self.device:isPocketBook() and not self.device:isAndroid() then
-                -- NOTE: PocketBook is a special snowflake, synthetic Power events are sent as EV_MSC.
-                --       Thankfully, that's all that EV_MSC is used for on that platform.
-                -- NOTE: Android, on the other hand, handles a *lot* of critical stuff over EV_MSC,
-                --       as it's used to communicate between Android and Lua land ;).
-                self._msc_ev_handler = self.handleMiscEv
-                self.handleMiscEv = self.voidEv
-            end
+        -- NOTE: We leave handleMiscEv alone, as EV_MSC is used for critical low-level stuff on some platforms (PB, Android),
+        --       and the only thing we might want to skip in there are gyro events anyway, which we'll handle separately.
```

Comment:
I would keep the specifics in as they are useful for grasping the scale of things.

Reply:
The previously more detailed comment, you mean?

Reply:
Yeah, knowing how much and for what something is used is often very helpful for deciding if you're going to touch it ;).